packages/react/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Let's just remove the README file for now until we get proper docs written

Reply:
done